conf/nginx.conf

```diff
@@ -1,23 +1,68 @@
 location /Microsoft-Server-ActiveSync {
```

Comment:
alias traversal here.
https://github.com/YunoHost/example_ynh/pull/45

Reply:
Here is different, it's not an alias, /Microsoft-Server-ActiveSync must be redirect to index.php without ending /

CF documentation: https://wiki.z-hub.io/display/ZP/Nginx+configuration

Reply:
Ok